src/lib/import/import-utils.ts

```diff
@@ -182,7 +182,12 @@ export class ImportUtils {
         roleRef.logic.delegate = this.resolveLogicValue(this.tagValue(xmlRoleRefLogic, 'delegate'));
         roleRef.logic.perform = this.resolveLogicValue(this.tagValue(xmlRoleRefLogic, 'perform'));
         /* @deprecated - 'this.resolveLogicValue(this.tagValue(xmlRoleRefLogic, 'assigned'))' is deprecated and it and following line will be removed in future versions. */
-        roleRef.logic.assign = this.resolveLogicValue(this.tagValue(xmlRoleRefLogic, 'assigned')) || this.resolveLogicValue(this.tagValue(xmlRoleRefLogic, 'assign'))
+        const assigned = this.resolveLogicValue(this.tagValue(xmlRoleRefLogic, 'assigned'));
+        if (assigned !== undefined) {
+            roleRef.logic.assign = assigned;
+        } else {
+            roleRef.logic.assign = this.resolveLogicValue(this.tagValue(xmlRoleRefLogic, 'assign'));
```

Comment:
based on lines 185 and 186 this call can set `roleRef.logic.assign` to `undefined`. Is this intended?